core/lib/zksync_core/src/api_server/execution_sandbox/execute.rs

```diff
@@ -108,40 +109,41 @@ pub(crate) async fn execute_tx_eth_call(
     vm_result
 }
 
-#[tracing::instrument(skip_all)]
-pub(crate) async fn execute_tx_with_pending_state(
-    vm_permit: VmPermit,
-    mut shared_args: TxSharedArgs,
-    execution_args: TxExecutionArgs,
-    connection_pool: ConnectionPool,
-    tx: Transaction,
-) -> (VmExecutionResultAndLogs, TransactionExecutionMetrics) {
-    let mut connection = connection_pool.access_storage_tagged("api").await.unwrap();
-    let block_args = BlockArgs::pending(&mut connection).await;
-    drop(connection);
-    // In order for execution to pass smoothlessly, we need to ensure that block's required gasPerPubdata will be
-    // <= to the one in the transaction itself.
-    shared_args.adjust_l1_gas_price(tx.gas_per_pubdata_byte_limit());
-
-    execute_tx_in_sandbox(
-        vm_permit,
-        shared_args,
-        execution_args,
-        connection_pool,
-        tx,
-        block_args,
-        vec![],
-    )
-    .await
-}
+// #[tracing::instrument(skip_all)]
+// pub(crate) async fn execute_tx_with_pending_state(
+//     vm_permit: VmPermit,
+//     mut shared_args: TxSharedArgs,
+//     execution_args: TxExecutionArgs,
+//     connection_pool: ConnectionPool,
+//     tx: Transaction,
+// ) -> (VmExecutionResultAndLogs, TransactionExecutionMetrics) {
+//     let mut connection = connection_pool.access_storage_tagged("api").await.unwrap();
+//     let block_args = BlockArgs::pending(&mut connection).await;
+//     drop(connection);
+//     // In order for execution to pass smoothlessly, we need to ensure that block's required gasPerPubdata will be
+//     // <= to the one in the transaction itself.
+//     shared_args.adjust_l1_gas_price(tx.gas_per_pubdata_byte_limit());
+
+//     execute_tx_in_sandbox(
+//         vm_permit,
+//         shared_args,
+//         execution_args,
+//         connection_pool,
+//         tx,
+//         block_args,
+//         vec![],
+//     )
+//     .await
+// }
```

Comment:
Why?

Reply:
to pass `block_args` more efficiently, but yeah, I'll def remove this commented out code